backend/rest/userRoutes.ts

```diff
@@ -203,4 +208,87 @@ userRouter.get(
   },
 );
 
+/* Facilitator will be able to view a list of all their learners */
+userRouter.get(
+  "/facilitator/myLearners",
```

Comment:
The ticket specified `GET /users/myLearners` and `PUT /users/myLearners/:userId`, but  `GET /users/myLearners` conflicts with  `GET /users/:userId` so this change was made

Reply:
good catch!!